cogs/fun.py

```
@@ -0,0 +1,83 @@
+            roast = random.choice(data[category])
+            await i.response.send_message(f'{roast}')
+        except Exception as e:
+            await i.response.send_message(f'{e}')
+
+
+
+    @nextcord.slash_command(name="dog", description="Posts a random dog picture from the dog api")
+    async def dog(self, i: Interaction):
+        try:
+            async with aiohttp.ClientSession() as session:
+                async with session.get('https://dog.ceo/api/breeds/image/random') as r:
```

Ruff (0.11.9)

Use a single `with` statement with multiple contexts instead of nested `with` statements

(SIM117)